kolibri_instant_schools_plugin/assets/src/views/account-page/index.vue

```diff
@@ -31,18 +33,18 @@
 
     <form @submit.prevent="submitEdits">
       <ui-alert
-        v-if="success"
+        v-if="nameSuccess"
```

Comment:
could you namespace these variables to something using "profile" (e.g. "profileName")? It makes it clearer that it's controlling the "profile" section of this page.